pegasus/sites.v3/code.org/public/ops-dashboard/index.haml

```diff
@@ -70,11 +71,11 @@ title: "Ops Dashboard"
   services.factory('programTypesService', [
     function() {
       var program_types = {
-        1: {id: 1, short_name: 'CS in Science', long_name: 'Computer Science in Science'},
-        2: {id: 2, short_name: 'CS in Algebra', long_name: 'Computer Science in Algebra'},
-        3: {id: 3, short_name: 'ECS', long_name: 'Exploring Computer Science'},
-        4: {id: 4, short_name: 'CSP', long_name: 'Computer Science Principles'},
-        5: {id: 5, short_name: 'K5', long_name: 'Computer Science in K5'}
+        1: {id: 1, short_name: "#{ ActivityConstants::PROGRAM_TYPES[1][:short_name]}", long_name: "#{ ActivityConstants::PROGRAM_TYPES[1][:long_name]}"},
```

Comment:
@philbogle 

I thought about doing a loop. Could be a good idea.

Re 0-based indexing, that wouldn't be a good idea now because there's already data that is mapped to the current indexes. I could probably subtract 1 from all of them and then change the mapping, but I'll need a little time to be extra careful and do it right.